applications/GeoMechanicsApplication/custom_processes/geo_apply_constant_scalar_value_process.cpp

```diff
@@ -8,219 +8,81 @@
 //  License:         geo_mechanics_application/license.txt
 //
 //  Main authors:    Riccardo Rossi
+//                   Richard Faasse
 //
 
 #include "geo_apply_constant_scalar_value_process.h"
-#include "containers/model.h"
+#include "containers/variable.h"
+#include "containers/variable_data.h"
+#include "includes/kratos_components.h"
+#include "includes/kratos_parameters.h"
+#include "includes/model_part.h"
 #include "utilities/variable_utils.h"
 
 namespace Kratos
 {
 
-KRATOS_CREATE_LOCAL_FLAG(GeoApplyConstantScalarValueProcess, VARIABLE_IS_FIXED, 0)
-
-GeoApplyConstantScalarValueProcess::GeoApplyConstantScalarValueProcess(Model& rModel, Parameters ThisParameters)
-    : GeoApplyConstantScalarValueProcess(
-          rModel.GetModelPart(ThisParameters["model_part_name"].GetString()), ThisParameters)
-{
-}
-
-GeoApplyConstantScalarValueProcess::GeoApplyConstantScalarValueProcess(ModelPart& rModelPart, Parameters ThisParameters)
-    : Process(Flags()), mrModelPart(rModelPart)
+GeoApplyConstantScalarValueProcess::GeoApplyConstantScalarValueProcess(ModelPart& rModelPart,
+                                                                       const Parameters& ThisParameters)
```

Comment:
Just to comply with the Kratos Style Guide:
```suggestion
                                                                       const Parameters& rParameters)
```

Reply:
Done